docker/dockerfiles/Dockerfile

```diff
@@ -44,19 +44,19 @@ RUN apt-get update && \
       libssl-dev \
       curl \
       libreadline-dev \
-      python python-dev \
+      python3 python3-dev \
```

Comment:
this might be a significant side effect; manually installing nodejs the way we do requires python. Currently, we're using python 2 but node 16 requires python 3:

```
Please use python3.11 or python3.10 or python3.9 or python3.8 or python3.7 or python3.6.
Node.js configure: Found Python 2.7.17...
The command '/bin/sh -c wget https://nodejs.org/dist/v16.19.0/node-v16.19.0.tar.gz &&     tar -xzvf node-v16.19.0.tar.gz &&     rm node-v16.19.0.tar.gz &&     cd node-v16.19.0 &&     ./configure &&     make -j4 &&     make install &&     cd .. &&     rm -r node-v16.19.0' returned a non-zero code: 1
```

I'm not sure what else (if anything) we use python for, and whether we should perhaps install both 2 and 3 simultaneously here.